assets/webpack.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   },
   plugins: [
     new webpack.DefinePlugin({
-      __PACKAGE_VERSION__: JSON.stringify(package.version)
+      PACKAGE_VSN: JSON.stringify(package.version)
```

Comment:
Actually, I had an idea. Can we make it so PACKAGE_VSN here is only the first two parts? I.e. instead of "0.14.1", it will be "0.14". I was thinking about doing it on the server but I think we can precompute it both on the client and server.